src/bsp/shared/src/bsp_default_symtab.c

```diff
@@ -29,8 +29,9 @@
  *   which will override this default.
  */
 
-#include "osapi.h"
 #include "bsp-impl.h"
 
+#include <osapi-module.h>
```

Comment:
Double quotes per coding standard?  Also all other occurrences...

Reply:
Fixed in rebase - now uses double quotes.